src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -1100,14 +1069,11 @@ public override EtlProcessProgress GetProgress(DocumentsOperationContext documen
                 if (result.NumberOfDocumentTombstonesToProcess > 0)
                     result.NumberOfDocumentTombstonesToProcess -= performance.NumberOfTransformedTombstones[EtlItemType.Document];
 
-                if (result.NumberOfCountersToProcess > 0)
-                    result.NumberOfCountersToProcess -= performance.NumberOfTransformedItems[EtlItemType.Counter];
-
-                if (result.NumberOfCounterTombstonesToProcess > 0)
-                    result.NumberOfCounterTombstonesToProcess -= performance.NumberOfTransformedTombstones[EtlItemType.Counter];
+                if (result.NumberOfCounterGroupsToProcess > 0)
+                    result.NumberOfCounterGroupsToProcess -= performance.NumberOfTransformedItems[EtlItemType.Counter];
```

Comment:
Here we have counter group vs number of particular items in a group, don't we? E.g. we have a one group but we processed 2 counters from that group, possible?

If it's gonna work differently then maybe renaming `EtlItemType.Counter` to `EtlItemType.CounterGroup` would make more sense?

Reply:
yes `EtlItemType.CounterGroup` makes more sense